content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+through a single Istio virtual service.
+
+The following example shows an Istio virtual service configuration file with
+descriptive value entries. You can use this example as a basis and modify it to
+fit your needs:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-vtl-svc
+spec:
+  hosts:
+    - *.my-co.org
+      route:
+      - destination:
+        host: service
+        subset: v1
+{{< /text >}}
+
+Virtual services can specify the destination's DNS host name using the `hosts:`
+field under `spec:`. As you can see in the `my-vtl-svc` example, you can use a
+wildcard for the virtual service configurations to apply to multiple hosts. In
+the `destination:` field of each route, you specify the service and subset that
+the configuration targets. The example configures a route for traffic to reach
+the `v1` subset of the `my-svc` service from the `my-vtl-svc` virtual service.
+The following diagram shows the configured rule:
+
+{{< image width="50%"
+    link="./net-config-3.svg"
+    caption="A virtual service configurable traffic to a specific subset"
+    >}}
+
+Istio does not need virtual services to connect to services. More commonly, you
+use virtual services to finely configure traffic. For example, to direct HTTP
+traffic to use a different version of the service for a specific user. To
+specify the user, the following example uses a regular expression to match the
+user from the cookie and shows you how to add the condition for the protocol in
+the `http:` field and a new `route:` and `destination:` configuration for the
+traffic matching the condition. In short, the example configures the HTTP
+traffic from the `jason` user to go to the `v2` subset of the `my-svc` service:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-vtl-svc
+spec:
+  hosts:
+    - *
+    - route:
+      - destination:
+        host: my-svc
+        subset: v1
+  http:
+    - match:
+      - headers:
+          cookie:
+            regex: "^(.*?;)?(user=jason)(;.*)?$"
+    route:
+      - destination:
+        host: my-svc
+        subset: v2
+{{< /text >}}
+
+The following diagram shows the configured traffic rules for both the traffic
+with the matched cookie and for all other traffic:
+
+{{< image width="50%"
+    link="./net-config-4.svg"
+    caption="Configurable traffic rules for traffic with and without a matched cookie"
+    >}}
+
+If you are working with a Kubernetes-based mesh, use virtual services to manage
+the traffic to your namespaces. The following example shows the `my-namespace`
+virtual service. It configures traffic routes for the two services behind the
+`my-namespace.svc.cluster.local` Kubernetes namespace:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-namespace
+spec:
+  hosts:
+    - *.my-namespace.svc.cluster.local
```

Comment:
@frankbu Please correct me if I'm wrong but, the way this is currently configured, means:
1. Only requests intended for the `my-namespace.svc.cluster.local` namespace will follow the route in this virtual service regardless of whether they are intended for svc1 or svc2.
2. Only requests intended for the `svc1.my-namespace.svc.cluster.local` follow the route to `host: svc-1`
3. Only requests intended for the `svc2.my-namespace.svc.cluster.local` follow the route to `host: svc-2`
Just to reiterate, this doc is trying to explain things, I'm aware that out in the wild developers probably create a virtual service for each pod if they need granular traffic control to them instead of doing it on a namespace level.

Reply:
@oaktowner You can see the flow and how the hosts are intended to be resolved. IMU (In My Understanding), he wildcard in `hosts:` allow configured virtual service to apply to any pods (svc-X) in the `my-namespace.svc.cluster.local` namespace. Then the specific routes are configured for individual pods. The prefix is used because what distinguishes one pod's name from the others is what comes before.